src/seedsigner/models/seed.py

```diff
@@ -164,6 +165,38 @@ def get_bip85_child_mnemonic(self, bip85_index: int, bip85_num_words: int, netwo
         return bip85.derive_mnemonic(root, bip85_num_words, bip85_index)
         
 
+    # ----------------- BIP-352 Silent Payments support -----------------
```

Comment:
I think these methods are fine here for now. But in general we have pretty fuzzy distinctions between what operations are in `Seed` vs `embit_utils` and ultimately what we hope to be in `embit` itself.

These feel like they should eventually be in `embit`.